static/js/compose_actions.js

```diff
@@ -396,6 +396,10 @@ exports.on_narrow = function () {
     }
 
     if (narrow_state.narrowed_by_pm_reply()) {
+        // If force_close is an option do not open compose box
```

Comment:
The rare case when this would called if the compose box was already open if the compose box is not in focus. In that case, the unread PM would open, and keep the compose box unfocused in the same state as it was before. So we can continue cycling through the PMs without issue.

Reply:
Oh, yeah, I guess if you hit the "p" key, the compose box will always be closed to start with.  I think we could still handle this more generally.

~~~ js

387 exports.on_narrow = function (opts) {
388     if (opts.force_close) {
389         exports.cancel();
390         return;
391     }

Reply:
@showell  Yes that is better handling. Amended the commit with this addition and pushed.